tests/test_model.py

```diff
@@ -522,3 +522,117 @@ def test_model_diskonlyfilemodel_field_is_constructed_correctly(
         assert relevant_field == input
     else:
         assert relevant_field.filepath == input
+
+
+@pytest.mark.parametrize(
+    "input_process, expected_process_format",
+    [
+        pytest.param(1, "0"),
+        pytest.param(2, "0 1"),
+        pytest.param(3, "0 1 2"),
+        pytest.param(4, "0 1 2 3"),
+        pytest.param(5, "0 1 2 3 4"),
+    ],
+)
+def test_dimr_with_fmcomponent_saving_process(
+    tmp_path, input_process: int, expected_process_format: str
+):
+    component = FMComponent(
+        name="test",
+        workingDir=".",
+        inputfile="test.mdu",
+        process=input_process,
```

Comment:
Currently `process` is (incorrectly??) typehinted as an `Optional[str]`, but here you pass an `int`

Reply:
0 1 2 3  on itself cannot be an int right, which means it should be a string. or a list of ints?

Reply:
_set_process_correctly will return the string